### DISCUSSION TITLE
DESTEK Bölümü

### DISCUSSION THREAD

**Discussion by derebey278** [community/community#40229] [Other Feature Feedback, Questions, & Ideas]
<!DOCTYPE html>
<html lang="tr">
<head>
    <meta charset="UTF-8">
    <title>AHMET GÜMÜŞ</title>
   <script src="js/jquery-3.6.1.min.js"></script>
    <link rel="stylesheet" href="main.css">
    
<link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-rbsA2VBKQhggwzxH7pPCaAqO46MgnOM80zW1RWuH61DGLwZJEdK2Kadq2F9CUG65" crossorigin="anonymous">
<style>
    .row{
    border:2px solid transparent !important;
    margin:20px !important; 
    padding:15px !important;     border-radius:5px !important;
    box-shadow: 5px 5px 5px 5px gray !important;
}input:submit{
    background: black !important;
}
</style>
</head>
<body>
 
 <div class="container">
     <div class="row" style="">
         <center><h2><marquee behavior="500px" direction="500px">DESTEK TALEBİ OLUŞTUR</marquee></h2></center>
         <form action="">
            <div class="form-group">
                 <label for="adsoyad">Ad Soyad </label>
                 <input id="adsoyad" type="text" placeholder="Ad Soyad" name="adsoyad" class="form-control"></input>
                <smal class="text-muted text-danger form-text">lütfen bu alanı doldurunuz</smal>
             </div><br>
            <div class="form-group">
                 <label for="konu">Konu </label>
                 <input id="konu" type="text" placeholder="Konu" name="Konu" class="form-control"></input>
                <smal class="text-muted text-danger form-text">lütfen bu alanı doldurunuz</smal>
             </div><br>
        
            <div class="form-group">
                 <label for="email">E-posta </label>
                 <input id="email" type="email" placeholder="E-posta " name="eposta " class="form-control"></input>
                <smal class="text-muted text-danger form-text">lütfen bu alanı doldurunuz</smal>
             </div><br>
            <div class="form-group">
                 <label for="mesaj">Mesaj </label>
                 <textarea id="mesaj" placeholder="Mesaj" name="mesaj" class="form-control"></textarea>
                <smal class="text-muted text-danger form-text">lütfen bu alanı doldurunuz</smal>
             </div> <br>
            <input type="submit" class="btn bg-dark text-white" value="Destek Talebi Oluştur">
         </form>
     </div>
 </div>
 
 
 

<script src="https://cdn.jsdelivr.net/npm/bootstrap@5.2.3/dist/js/bootstrap.bundle.min.js" integrity="sha384-kenU1KFdBIe4zVF0s0G1M5b4hcpxyD9F7jL+jjXkk+Q2h455rYXK/7HAuoJl+0I4" crossorigin="anonymous"></script>
   <script src="main.js"> </script>
</body>
</html>

**Comment by Akash1134** [comment]
Hi @derebey278  , thanks for participating!

We take our Code of Conduct very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
